shardingsphere-kernel/shardingsphere-parser/shardingsphere-parser-core/src/test/java/org/apache/shardingsphere/parser/yaml/swapper/YamlSQLParserRuleConfigurationSwapperTest.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.parser.yaml.swapper;
+
+import org.apache.shardingsphere.parser.config.SQLParserRuleConfiguration;
+import org.apache.shardingsphere.parser.constant.SQLParserOrder;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertTrue;
+
+import org.apache.shardingsphere.parser.yaml.config.YamlSQLParserCacheOptionRuleConfiguration;
+import org.apache.shardingsphere.parser.yaml.config.YamlSQLParserRuleConfiguration;
+import org.apache.shardingsphere.sql.parser.api.CacheOption;
+import org.junit.Test;
+
+public final class YamlSQLParserRuleConfigurationSwapperTest {
+    
+    @Test
+    public void assertSwapToYamlConfiguration() {
+        YamlSQLParserRuleConfiguration actualResult =
+                new YamlSQLParserRuleConfigurationSwapper().swapToYamlConfiguration(new SQLParserRuleConfiguration(true, new CacheOption(2, 5), new CacheOption(4, 7)));
+        assertTrue(actualResult.isSqlCommentParseEnabled());
+        assertThat(actualResult.getParseTreeCache().getInitialCapacity(), is(2));
+        assertThat(actualResult.getParseTreeCache().getMaximumSize(), is(5L));
+        assertThat(actualResult.getSqlStatementCache().getInitialCapacity(), is(4));
+        assertThat(actualResult.getSqlStatementCache().getMaximumSize(), is(7L));
+    }
+    
+    @Test
+    public void assertSwapToObjectWithDefaultConfig() {
+        YamlSQLParserRuleConfiguration configuration = new YamlSQLParserRuleConfiguration();
+        configuration.setSqlCommentParseEnabled(true);
+        SQLParserRuleConfiguration actualResult = new YamlSQLParserRuleConfigurationSwapper().swapToObject(configuration);
+        assertThat(actualResult.getParseTreeCache().getInitialCapacity(), is(128));
```

Comment:
Can you rename actualResult to actual?

Reply:
done, https://github.com/apache/shardingsphere/pull/21290/commits/8daffcc452be4a3d21369cf19a5409845431f139